container/Dockerfile.cpu

```diff
@@ -0,0 +1,56 @@
+FROM ubuntu:18.04
+ARG APP=xgboost
+
+ENV PYTHONUNBUFFERED TRUE
+
+# Disable thread pinning in TVM and Treelite
+ENV TVM_BIND_THREADS 0
+ENV TREELITE_BIND_THREADS 0
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    build-essential \
+    python3 \
+    python3-pip \
+    openjdk-8-jdk-headless \
+    curl \
+    git \
+    cmake \
+    && rm -rf /var/lib/apt/lists/* \
+    && cd /tmp
+
+RUN pip3 install --pre --no-cache-dir mxnet-model-server
+
+RUN pip3 install --no-cache-dir numpy scipy xlrd Pillow
+
+#RUN pip3 install --no-cache-dir dlr
+
+RUN mkdir -p /home/model-server && cd /home/model-server \
+    && git clone --recursive https://github.com/neo-ai/neo-ai-dlr \
+    && cd neo-ai-dlr && git checkout 7dd33f829062e19df3c0f175912199ad176da970 \
```

Comment:
why `7dd33f829062e19df3c0f175912199ad176da970`?

Reply:
@wweic Let's keep it for now. I'll submit a follow-up PR where the Docker build will embed the latest DLR artifact into the container.

Reply:
OK. Please create an issue and follow up with a PR.

Reply:
See #43